### PR TITLE
Adding Allium to Biolab

### PR DESCRIPTION
**Pull request by Akiyama64**
This is a copy of this pull https://github.com/ttk2/FactoryMod/pull/85

In honor of this thread http://www.reddit.com/r/Civcraft/comments/35etth/alliums/ , since alliums and flower forests do not exist on Civcraft, let's make a recipe for it in the Biolab, please.

### CODE REVIEW

**Comment by ttk2** [comment]
is this still needed?

**Comment by Akiyama64** [comment]
Well, there's no other source for alliums, so this pull seems to still be needed.

**Comment by WildWeazel** [comment]
The red sand recipe will need to include at least 1 sand input per red sand output, otherwise glass will be renewable

**Comment by ttk2** [comment]
Is that handled in this Travis?

**Comment by Akiyama64** [comment]
"Is that handled in this Travis?"

This pull is only for one flower, the allium.

**Comment by ribagi** [comment]
Did you see if this was valid yaml?

**Comment by ttk2** [comment]
its on Civtest, no errors.

**Comment by ttk2** [comment]
never mind, its invalid.

org.bukkit.configuration.InvalidConfigurationException: mapping values are
not allowed here
│·········································
 in 'string', line 7444, column 12:

│·········································
         inputs:

 │·········································
               ^

 │·········································


│·········································
        at
org.bukkit.configuration.file.YamlConfiguration.loadFromString(YamlConfiguration.java:57)
~[spigot.jar:git-Spigot-1d14d5f-7│·········································
722428]

│·········································
        at
org.bukkit.configuration.file.FileConfiguration.load(FileConfiguration.java:226)
~[spigot.jar:git-Spigot-1d14d5f-7722428]
 │·········································
        at
org.bukkit.configuration.file.FileConfiguration.load(FileConfiguration.java:169)
~[spigot.jar:git-Spigot-1d14d5f-7722428]
 │·········································
        at
org.bukkit.configuration.file.YamlConfiguration.loadConfiguration(YamlConfiguration.java:180)
[spigot.jar:git-Spigot-1d14d5│·········································
f-7722428]

 │·········································
        at
org.bukkit.plugin.java.JavaPlugin.reloadConfig(JavaPlugin.java:188)
[spigot.jar:git-Spigot-1d14d5f-7722428]
 │·········································
        at org.bukkit.plugin.java.JavaPlugin.getConfig(JavaPlugin.java:162)
[spigot.jar:git-Spigot-1d14d5f-7722428]
│·········································
        at
com.github.igotyou.FactoryMod.FactoryModPlugin.initConfig(FactoryModPlugin.java:122)
[FactoryMod-1.3.4-97.jar:?]
│·········································
        at
com.github.igotyou.FactoryMod.FactoryModPlugin.onEnable(FactoryModPlugin.java:90)
[FactoryMod-1.3.4-97.jar:?]
 │·········································
        at
org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:321)
[spigot.jar:git-Spigot-1d14d5f-7722428]
 │·········································
        at
org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:335)
[spigot.jar:git-Spigot-1d14d5f-7722428]
 │·········································
        at
org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:405)
[spigot.jar:git-Spigot-1d14d5f-7722428]
│·········································
        at
org.bukkit.craftbukkit.v1_8_R2.CraftServer.loadPlugin(CraftServer.java:356)
[spigot.jar:git-Spigot-1d14d5f-7722428]
 │·········································
        at
org.bukkit.craftbukkit.v1_8_R2.CraftServer.enablePlugins(CraftServer.java:316)
[spigot.jar:git-Spigot-1d14d5f-7722428]
│·········································
        at
net.minecraft.server.v1_8_R2.MinecraftServer.r(MinecraftServer.java:416)
[spigot.jar:git-Spigot-1d14d5f-7722428]
│·········································
        at
net.minecraft.server.v1_8_R2.MinecraftServer.k(MinecraftServer.java:382)
[spigot.jar:git-Spigot-1d14d5f-7722428]
│·········································
        at
net.minecraft.server.v1_8_R2.MinecraftServer.a(MinecraftServer.java:337)
[spigot.jar:git-Spigot-1d14d5f-7722428]
│·········································
        at
net.minecraft.server.v1_8_R2.DedicatedServer.init(DedicatedServer.java:257)
[spigot.jar:git-Spigot-1d14d5f-7722428]
 │·········································
        at
net.minecraft.server.v1_8_R2.MinecraftServer.run(MinecraftServer.java:522)
[spigot.jar:git-Spigot-1d14d5f-7722428]
│·········································
        at java.lang.Thread.run(Thread.java:745) [?:1.8.0_45-internal]

File: config.yml [hunk 1]
```diff
@@ -7427,6 +7428,21 @@ production_recipes:
...
+  Sequencing_Allium:
+    name: Sequencing Allium
+     inputs:
+      Poppy:
+        material: RED_ROSE
+         amount: 8
+      Blue Orchid:
+        material: RED_ROSE
+         amount: 8
+        durablity: 1
```
Comment (Akiyama64) [inline config.yml]: I misspelled durability.

Comment (ttk2) [inline config.yml]: Just make a pull for a fix. No biggie.

**Pull request merged**